wlog/auditlog/audit3log/categories_entity_extractors.go

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) 2025 Palantir Technologies. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package audit3log
+
+import (
+	"github.com/palantir/pkg/rid"
+	commonv2 "github.com/palantir/witchcraft-go-logging/conjure/foundry/audit/api/common/v2"
+	categoriespkg "github.com/palantir/witchcraft-go-logging/wlog/auditlog/audit3log/internal/categories"
+)
+
```

Comment:
Checked Java definition/implementation, and currently there are no custom entity extractors (all entity extractors use the default logic), although the visitor logic does allow for adding a custom extractor.

In Go, kept this logic structured in the same manner as UID extractor, so if a category that is added in the future were to need a custom entity extractor, it could be set up in an `init` function here in the same manner as for UID extractors.